manager/assets/modext/core/modx.layout.js

```diff
@@ -52,13 +50,22 @@ MODx.Layout = function(config){
 };
 Ext.extend(MODx.Layout, Ext.Viewport, {
     /**
+     * Initialize the state provider to use
+     */
+    initStateProvider: function() {
+        var provider = MODx.config.stateprovider || MODx.HttpProvider,
+            sp = new provider();
+        Ext.state.Manager.setProvider(sp);
+        sp.initState(MODx.defaultState);
```

Comment:
`initState` is bound to the `MODx.HttpProvider` not a method shipped with [`Ext.state.Provider`](http://docs.sencha.com/extjs/3.4.0/#!/api/Ext.state.Provider), so we might want to make sure the method/function exists before calling it.

While being there, we might also want to add an option to disable the manager controllers to check for the default states (see `modManagerController::getDefaultState`)